codingTest_김준우/자릿수 더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+  const strN = String(n);
+  let splitN = strN.split("");
```

Comment:
```suggestion
  const splitN = strN.split("");
```
splitN은 변경되지 않는 상수이기 때문에 const로 선언해주시면 되겠습니다. Number(splitN[i])를 한다고 해서 splitN의 값이 변하는게 아니기 때문입니다.